lib/redis.js

```diff
@@ -0,0 +1,25 @@
+export default class Redis {
+  constructor (options) {
+    this._redis = require('redis').createClient(options.url )
+  }
+
+  createHealthCheck () {
+    return (callback, that = this) => {
```

Comment:
en fait c'est un peu bof ca, car si je file la reference de la fonction, alors la fonction ne marche plus car elle n'a plus accès a this.
Du coup je vais regader bind